<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>grafi_empleados</class>
 <widget class="QWidget" name="grafi_empleados">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>201</width>
     <height>282</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="pbtn_previous">
          <property name="text">
           <string>&lt;&lt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbtn_next">
          <property name="text">
           <string>&gt;&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Nombre</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_name"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>DNI</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_dni"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Edad</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_email"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Genero</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_telf"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>direcci√≥n</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_dir"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pbtn_add">
      <property name="text">
       <string>ADD</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pbtn_nuevo">
      <property name="text">
       <string>Nuevo</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
